### DISCUSSION TITLE
In English, "me" is being pronounced as "muh"

### DISCUSSION THREAD

**Discussion by markunas** [deepgram/community#682] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - TTS

### Details

At least two voices (aura-asteria-en & aura-luna-en) are pronouncing the word "me" in English as "muh"

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

https://api.deepgram.com/v1/speak

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by markunas** [comment]
This appears to be fixed now. Thank you.

**Comment by markunas** [comment]
Reopening. Pronouncing "Me" as "Muh" is happening again.

**Comment by jpvajda** [comment]
@markunas  this could be possibly one of those situations where the model is "hallucinating" a bit. 😄 

* Have you tried another model? 
* Which model are using where you are seeing this behavior?
* Can you give us a sample of the text you used with the word Me in it?


We have [12 TTS models currently ](https://developers.deepgram.com/docs/tts-models)and one might be better than another at properly handling this word in a text input, I'll also let one of our Engineering Teams know this is occurring for you consistently.

**Comment by markunas** [comment]
@jpvajda I do believe it is hallucinating, but to hallucinate on such a common word as "me" in English is troublesome.

* We are experiencing this with the 2 models we tested: `aura-asteria-en` & `aura-luna-en`
* Here's the sample text where we experience this: "Hey, my name is `<name>`, and I'm an automated agent calling on a recorded line. Sometimes it might take me a second to respond." >> The word "me" here is pronounced as "muh"

Thanks for your help!

**Comment by jpvajda** [comment, marked as answer]
@markunas  thanks for the addition details here. I'll pass this info along to our Engineering Team who is currently investigating this issue.


If you could would you be able share the name you are using when the issue happens?
